testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/OperationWarningTestsCase.java

```diff
@@ -0,0 +1,131 @@
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.REMOVE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.NAME;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.VALUE;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OPERATION_HEADERS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.WRITE_ATTRIBUTE_OPERATION;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESPONSE_HEADERS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.WARNINGS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.WARNING;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.LEVEL;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESULT;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.controller.logging.ControllerLogger;
+import org.jboss.as.remoting.logging.RemotingLogger;
+import org.jboss.as.test.integration.domain.management.util.DomainLifecycleUtil;
+import org.jboss.as.test.integration.domain.management.util.DomainTestSupport;
+import org.jboss.dmr.ModelNode;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationWarningTestsCase {
+
+    private static DomainTestSupport testSupport;
+    private static DomainLifecycleUtil domainMasterLifecycleUtil;
+    private static DomainLifecycleUtil domainSlaveLifecycleUtil;
+
+    protected static final String NAME_WORKER = "puppet-master";
+    protected static final String WORKER = "worker";
+    protected static final String NAME_PROFILE = "default";
+    protected static final PathAddress ADDRESS_WORKER = PathAddress.pathAddress(PathElement.pathElement(PROFILE, NAME_PROFILE),
+            PathElement.pathElement(SUBSYSTEM, "io"), PathElement.pathElement(WORKER, NAME_WORKER));
+    protected static final PathAddress ADDRESS_REMOTING = PathAddress.pathAddress(
+            PathElement.pathElement(PROFILE, NAME_PROFILE), PathElement.pathElement(SUBSYSTEM, "remoting"),
+            PathElement.pathElement("configuration", "endpoint"));
+    protected static final String BAD_LEVEL = "X_X";
+
+    @Before
+    public void before() throws Exception {
+        testSupport = DomainTestSuite.createSupport(DatasourceTestCase.class.getSimpleName());
+        domainMasterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
+        domainSlaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
```

Comment:
This stuff should be in a @BeforeClass with reversal in an @AfterClass. The addWorker()/setRemotingWorkerTo("default", "OFF")/removeWorker() can remain in a @Before/@After although since there's only 1 test method it's fine to put them in the @BeforeClass/@AfterClass too